data/phi0119/phi006/__cts__.xml

```diff
@@ -1,11 +1,11 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0119.phi006" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0119">
             <ti:title xml:lang="lat">Casina</ti:title>
             <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0119.phi006.perseus-eng2" workUrn="urn:cts:latinLit:phi0119.phi006">
-              <ti:label xml:lang="eng">Casina, The comedies of Plautus Vol II</ti:label>
+              <ti:label xml:lang="eng">Casina, or The Stratagem Defeated</ti:label>
```

Comment:
you used "multiple" elsewhere

Reply:
I just saw those I will fix. And you are quite right @lcerrato this is a much more efficient way to get reviews.